filetests/isa/x86/optimized-zero-constants.cton

```diff
@@ -0,0 +1,9 @@
+test binemit
+isa x86
```

Comment:
Currently in cretonne, `x86` defaults to 32-bit x86. Since the encoding.py code below only handles 64-bit x86 at the moment, it'd be good to add "set is_64bit=1" here so that we can get closer to plumbing this feature all the way through, and we can go back and add 32-bit support later.